app/javascript/courses/curricula/components/CoursesCurriculum__LevelSelector.re

```diff
@@ -15,82 +15,102 @@ let levelSelectorClasses = isSelected => {
   );
 };
 
-let updateSelectedLevel =
-    (levels, setSelectedLevelId, showLevelZero, setShowLevelZero, event) => {
-  let selectedLevelId = ReactEvent.Form.target(event)##value;
-  let level =
-    levels |> ListUtils.findOpt(l => l |> Level.id == selectedLevelId);
-
-  switch (level) {
-  | Some(_) =>
-    showLevelZero ? setShowLevelZero(false) : ();
-    setSelectedLevelId(selectedLevelId);
-  | None => ()
-  };
-};
-
 let levelName = level =>
   "L"
   ++ (level |> Level.number |> string_of_int)
   ++ ": "
   ++ (level |> Level.name);
 
+let numberedLevelSelector = (selectedLevel, setShowLevelZero) => {
+  let defaultClasses = "rounded-l-lg font-semibold focus:outline-none cursor-pointer ";
+
+  let additionalClasses =
+    switch (setShowLevelZero) {
+    | Some(_) => "w-1/2 bg-white px-4 py-2 hover:bg-gray-100 hover:text-primary-500 truncate leading-loose text-sm"
+    | None => "w-full focus:shadow-inner bg-primary-100 border-t border-b px-2 h-10 flex items-center justify-between"
```

Comment:
@harigopal Remove `focus:shadow-inner` and add  `rounded-lg`

`w-full rounded-lg bg-primary-100 border-t border-b px-2 h-10 flex items-center justify-between`